app/main.py

```diff
@@ -93,11 +97,62 @@ async def registerUser(request: Request,
     else:
         access_token_expires = timedelta(minutes=ACCESS_TOKEN_EXPIRE_MINUTES)
         access_token = create_access_token(
-        data={"sub": user.email}, expires_delta=access_token_expires)
+            data={"email": user.email}, expires_delta=access_token_expires)
+        add_token_to_user(user.email, access_token)
         print(access_token)
-        return templates.TemplateResponse("profile.html",{ "request": request })
+        
+        return JSONResponse(content={"status": "200", "message": "Registered successfully", "token": access_token})
+###
+
+
+### login google
+
+### setup
+COOKIE_AUTHORIZATION_NAME = "Authorization"
+COOKIE_DOMAIN = "localhost"
+PROTOCOL = "http://"
+FULL_HOST_NAME = "localhost"
+PORT_NUMBER = 8000
+CLIENT_ID = "829120611572-clp4nv8n5sjhvn7dh1oqq32egp9ef931.apps.googleusercontent.com"
 ###
 
+### paths 
+@app.get("/google_login_client", response_class=HTMLResponse)
```

Comment:
muta astea de aici si punele intr-un fisier separat le includem in main. In main ar fi ok sa ramana decat constanttel si partile main ale aplicatiei